src/main.ts

```
@@ -1,8 +1,52 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { MicroserviceOptions, Transport } from '@nestjs/microservices';
+import { join } from 'path';
 
 async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
-  await app.listen(process.env.PORT ?? 3000);
+  const app = await NestFactory.createMicroservice<MicroserviceOptions[]>(
+    AppModule,
+    [
+      { // TCP
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)